### PR TITLE
Add link Snap.js callback transaction result

### PR DESCRIPTION
**Pull request by Andrenzo17**
- update Snap Javascript Callback and add link to Snap Docs.

### CODE REVIEW

File: en/snap/advanced-feature.md [hunk 1]
```diff
@@ -218,6 +218,8 @@ window.snap.pay('SNAP_TRANSACTION_TOKEN', {
 ```
 <!-- tabs:end -->
 
+Example of the Snap.js callback transaction result and Snap.js callback parameters [please refer to Snap Docs](https://snap-docs.midtrans.com/#js-callback).
+
```
Comment (rizdaprasetya) [inline en/snap/advanced-feature.md:222]: hmm not the most accurate wording & grammar. Should be:
```
For more details on Snap JS callback's result JSON properties & the samples, refer to [this section of Snap Docs](https://snap-docs.midtrans.com/#js-callback)
```

File: en/technical-reference/docs-changelog.md [hunk 2]
```diff
@@ -3,6 +3,9 @@
 
 <!-- #### Unreleased 
 -->
+#### 2022/08/02
+- update Snap Javascript Callback and add link to Snap Docs.
+
```
Comment (rizdaprasetya) [inline en/technical-reference/docs-changelog.md:8]: > update Snap Javascript Callback

That's ambiguous, you are not updating the callback. You are just clarifying the docs. Should be:
> improve Snap JS callback advanced section by adding link to Snap Docs.

**Review by rizdaprasetya — CHANGES_REQUESTED** [review]

**Comment by Andrenzo17** [comment]
Thank you for Feedback kak, kindly to review again. 🙏

**Review by rizdaprasetya — APPROVED** [review]
Looks good now, thanks ndre 👍 , will merge

**Pull request merged**